backend/app/resources/create_invited_user_dto.py

```diff
@@ -0,0 +1,18 @@
+class CreateInvitedUserDTO:
+    def __init__(self, **kwargs):
+        self.email = kwargs.get("email")
+        self.role = kwargs.get("role")
+        self.first_name = kwargs.get("first_name")
+        self.last_name = kwargs.get("last_name")
+
+    def validate(self):
+        error_list = []
+        if type(self.email) is not str:
+            error_list.append("The email supplied is not a string.")
+        if type(self.role) is not str:
```

Comment:
instead of checking if this is a string, should we instead check if its one of the roles?